cmd/cosign/cli/verify.go

```diff
@@ -92,12 +93,15 @@ against the transparency log.`,
 				return err
 			}
 
+			if o.CertVerify.CertIdentity == "" || o.CertVerify.CertOidcIssuer == "" {
```

Comment:
I think we instead want this in the Exec functions, like https://github.com/sigstore/cosign/blob/1c15b03a6e9e47f5c85a8fd7ff98bf75e910df6a/cmd/cosign/cli/verify/verify.go#L78

You also need this in the other verify commands, for manifest, dockerfile, and verify-blob